rollup-bridge-contracts/README.md

```diff
@@ -19,6 +19,15 @@ Solidity smart contracts in the repository are used for proof verification and s
 1. [NilVerifier-Contract](./contracts/verifier/NilVerifier.sol) contains the [verify](./contracts/verifier/NilVerifier.sol#L9) proof logic
 2. NilVerifier contract is non-upgradeable and stateless contract
 
+### BridgeContracts
```

Comment:
"## BridgeContracts" - same header level as `NilRollup`

Reply:
add to the list above
```
1. NilRollup
2. NilVerifier
3. BridgeContracts
```

Reply:
@akokoshn updated the readme doc